shared/chat/conversation/list-area/normal/index.native.js

```diff
@@ -203,7 +203,6 @@ class ConversationList extends React.PureComponent<Props> {
             keyExtractor={this._keyExtractor}
             // Limit the number of pages rendered ahead of time (which also limits attachment previews loaded)
             windowSize={5}
-            removeClippedSubviews={true}
```

Comment:
removed because true by default?

Reply:
removed as this seemed problematic for a bunch of ppl and i wanted to see if it changed anything

Reply:
I think the default value is true, you may want to set it to `false` instead

Reply:
by default its not defined and it has a comment saying you shouldn't use it
https://github.com/facebook/react-native/blob/e4392b773ce5a17257c93c5c8c5ac097ca60eb88/Libraries/Lists/VirtualizedList.js#L192

Reply:
K that prop plumbs to the one in `ScrollView`, their docs are wrong
https://github.com/facebook/react-native/blob/e4392b773ce5a17257c93c5c8c5ac097ca60eb88/Libraries/Components/ScrollView/ScrollView.js#L531-L532
anyway sounds good

Reply:
still says 'Experimental' and i've just seen a bunch of github issues where ppl complained so i think lets turn it off and see if it makes any difference either way